calc.rb

```diff
@@ -0,0 +1,37 @@
+class Calc
+  def initialize (oper, n1, n2)
+    @@oper = oper
```

Comment:
Here, we can just use ```@oper``` instead of ```@@oper```. Check the notes for the difference between class variables (@@var_name) and instance variables (@var_name).